emperor/support_files/js/draw.js

```diff
@@ -77,6 +112,37 @@ define(['underscore', 'three'], function(_, THREE) {
     return line;
   }
 
+  /**
+   *
+   * Create a generic Arrow object (composite of a cone and line)
+   *
+   * @param {float[]} from The x, y and z coordinates where the arrow
+   * originates from.
+   * @param {float[]} to The x, y and z coordinates where the arrow points to.
+   * @param {integer} color Hexadecimal base that specifies the color of the
+   * line.
+   *
+   * @return {THREE.Object3D}
+   * @function makeArrow
+   */
+  function makeArrow(from, to, color) {
+    // TODO: Write tests for this function
```

Comment:
Is this something you'd like to do for this PR, or should this be noted as an issue?

Reply:
I've removed this comment, there are now tests for this function 👍